test/e2e/pages/move-detail.js

```diff
@@ -189,6 +192,20 @@ class MoveDetailPage extends Page {
       }
     }
   }
+
+  async checkUpdateLink(category, exists = true) {
+    const selector = this.nodes.getUpdateLink(category)
+    await t.expect(selector.exists).eql(exists)
+  }
+
+  async checkNoUpdateLink(category) {
```

Comment:
Same here, this doesn't really add any real abstraction. In a test where the link should not appear this feels clear enough:

```javascript
await t.expect(moveDetailPage.nodes.getUpdateLink('risk').exists).not.ok()
```

or 

```javascript
await t.expect(moveDetailPage.nodes.getUpdateLink('risk').exists).ok()
```

Reply:
The conversation will continue...